src/components/ActionsPanel/constants.ts

```diff
@@ -0,0 +1,209 @@
+    button: {
+      content: translateRaw('CLAIM_UNI_ACTION_BUTTON'),
+      to: UNISWAP_LINK,
+      external: true
+    },
+    category: ACTION_CATEGORIES.THIRD_PARTY
+  },
+  {
+    name: 'renew_ens',
+    heading: translateRaw('RENEW_ENS_ACTION_HEADING'),
+    icon: 'ensLogo',
+    filter: (state: StoreContextState) =>
+      state.ensOwnershipRecords.some((r) =>
+        isBefore(new Date(r.expiryDate), add(new Date(), { days: 60 }))
+      ),
+    body: [translate('RENEW_ENS_ACTION_BODY')],
+    priority: 30,
+    button: {
+      content: translateRaw('RENEW_ENS_ACTION_BUTTON'),
+      to: ROUTE_PATHS.ENS.path,
+      external: false
+    },
+    category: ACTION_CATEGORIES.THIRD_PARTY
+  },
+  {
+    name: 'buy_hw',
+    heading: translateRaw('BUY_HW_ACTION_HEADING'),
+    icon: 'ledger-icon',
+    body: [translate('BUY_HW_ACTION_BODY')],
+    filter: (state: StoreContextState) => !state.accounts.some((c) => isHardwareWallet(c.wallet)),
+    priority: 30,
+    button: {
+      content: translateRaw('BUY_HW_ACTION_BUTTON'),
+      to: EXT_URLS.LEDGER_REFERRAL.url,
+      external: true
+    },
+    category: ACTION_CATEGORIES.SECURITY
+  },
+  {
+    name: 'myc_membership',
+    heading: translateRaw('MYC_MEMBERSHIP_ACTION_HEADING'),
+    icon: 'membership',
+    body: [translate('MYC_MEMBERSHIP_ACTION_BODY')],
+    filter: (state: StoreContextState) => !state.isMyCryptoMember,
+    priority: 0,
+    button: {
+      content: translateRaw('MYC_MEMBERSHIP_ACTION_BUTTON'),
+      to: ROUTE_PATHS.MYC_MEMBERSHIP.path,
+      external: false
+    },
+    category: ACTION_CATEGORIES.SELF_LOVE
+  },
+  {
+    name: 'add_account',
+    heading: translateRaw('ADD_ACCOUNT_ACTION_HEADING'),
+    icon: 'experience',
+    body: [translate('ADD_ACCOUNT_ACTION_BODY')],
+    filter: (state: StoreContextState) => state.accounts.length < 3,
+    priority: 0,
+    button: {
+      content: translateRaw('ADD_ACCOUNT_ACTION_BUTTON'),
+      to: ROUTE_PATHS.ADD_ACCOUNT.path,
+      external: false
+    },
+    category: ACTION_CATEGORIES.MYC_EXPERIENCE
+  },
+  {
+    name: 'backup',
+    heading: translateRaw('BACKUP_ACTION_HEADING'),
+    icon: 'experience',
+    body: [translate('BACKUP_ACTION_BODY')],
+    filter: (state: StoreContextState) => state.accounts.length >= 3,
+    priority: 0,
+    button: {
+      content: translateRaw('BACKUP_ACTION_BUTTON'),
+      to: ROUTE_PATHS.SETTINGS_EXPORT.path,
+      external: false
+    },
+    category: ACTION_CATEGORIES.MYC_EXPERIENCE
+  },
+  {
+    name: 'feedback',
+    heading: translateRaw('FEEDBACK_ACTION_HEADING'),
+    icon: 'feedback',
+    body: [translate('FEEDBACK_ACTION_BODY_1'), translate('FEEDBACK_ACTION_BODY_2')],
+    priority: 0,
+    button: {
+      content: translateRaw('FEEDBACK_ACTION_BUTTON'),
+      to: formatSupportEmail(translateRaw('FEEDBACK_ACTION_MAIL_SUBJECT')),
+      external: true
+    },
+    category: ACTION_CATEGORIES.OTHER
+  },
+  {
+    name: 'newsletter',
```

Comment:
How will tasks like this be marked as complete?

Reply:
They will never be marked as complete. It's part of the feature, Taylor wanted some actions that never completes.